Source/WebKit/UIProcess/mac/WebPageProxyMac.mm

```diff
@@ -215,7 +215,7 @@ static inline bool expectsLegacyImplicitRubberBandControl()
     
     const Seconds messageTimeout(20);
     auto sendResult = sendSync(Messages::WebPage::GetStringSelectionForPasteboard(), messageTimeout);
-    auto [value] = sendResult.takeReplyOr(String { });
+    auto [value] = sendResult.takeReplyOr(sendResult.defaultReplyArguments);
```

Comment:
is the `defaultReplyArguments` in general useful?
If it is only used in this pattern `takeReply(..defaultReplyArguments)`, would it be simpler to have:
```
auto [value] = sendResult.takeReplyOrDefaults();
```